### PR TITLE
fix: make Kotlin daemon args list consistent

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

File: docs/topics/gradle.md [hunk 1]
```diff
@@ -1008,15 +1008,16 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
 
 ### Setting Kotlin daemon's JVM arguments
 
-Each of the options in the following list overrides the ones that came before it:
-* If nothing is specified, the Kotlin daemon inherits arguments from the Gradle daemon.
+Each of the way to set arguments in the following list overrides the ones that came before it:
```
Comment (PetrakovichVictoria) [inline docs/topics/gradle.md:1011]:
```suggestion
Each of the ways to set arguments in the following list overrides the ones that came before it:
```

**Comment by danil-pavlov** [comment]
I don't think we use double underscores for bold. Alternatively, you can use headers of the next level. It'll also eliminate the need to use a sublist in the second item

**Review by danil-pavlov — COMMENTED** [review]
I don't think we use double underscores for bold. Alternatively, you can use headers of the next level. It'll also eliminate the need to use a sublist in the second item

**Comment by PetrakovichVictoria** [comment]
@danil-pavlov 1. yes, true 2. this is the good idea, thanks!

**Comment by PetrakovichVictoria** [comment]
I've made a list with clickable headers:

Screenshot 2022-10-05 at 19 41 21

Can also be seen on the staging.

File: docs/topics/gradle.md [hunk 2]
```diff
@@ -1008,89 +1008,104 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
...
-  > * Gradle is using JDK 1.9 or higher.
-  > * The version of Gradle is between 7.0 and 7.1.1 inclusively.
-  > * Gradle is compiling Kotlin DSL scripts.
-  > * There is no running Kotlin daemon.
-  > 
-  > To overcome this, upgrade Gradle to the version 7.2 (or higher) or use the `kotlin.daemon.jvmargs` property – see the following item.
-  >
-  {type="warning"}
+Each of the following ways to set arguments overrides the ones that came before it:
+* [Gradle daemon arguments inheritance](#gradle-daemon-arguments-inheritance).
```
Comment (danil-pavlov) [inline docs/topics/gradle.md:1012]:
```suggestion
* [Gradle daemon arguments inheritance](#gradle-daemon-arguments-inheritance)
```

File: docs/topics/gradle.md [hunk 3]
```diff
@@ -1008,89 +1008,104 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
...
+* [`kotlin.daemon.jvm.options` system property](#kotlin-daemon-jvm-options-system-property).
```
Comment (danil-pavlov) [inline docs/topics/gradle.md:1013]:
```suggestion
* [`kotlin.daemon.jvm.options` system property](#kotlin-daemon-jvm-options-system-property)
```

File: docs/topics/gradle.md [hunk 4]
```diff
@@ -1008,89 +1008,104 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
...
+* [`kotlin.daemon.jvmargs` property](#kotlin-daemon-jvmargs-property).
```
Comment (danil-pavlov) [inline docs/topics/gradle.md:1014]:
```suggestion
* [`kotlin.daemon.jvmargs` property](#kotlin-daemon-jvmargs-property)
```

File: docs/topics/gradle.md [hunk 5]
```diff
@@ -1008,89 +1008,104 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
...
+* [`kotlin` extension](#kotlin-extension).
```
Comment (danil-pavlov) [inline docs/topics/gradle.md:1015]:
```suggestion
* [`kotlin` extension](#kotlin-extension)
```

File: docs/topics/gradle.md [hunk 6]
```diff
@@ -1008,89 +1008,104 @@ The Kotlin daemon uses the same JDK that the Gradle daemon does.
...
+* [Specific task definition](#specific-task-definition).
```
Comment (danil-pavlov) [inline docs/topics/gradle.md:1016]:
```suggestion
* [Specific task definition](#specific-task-definition)
```

**Review by danil-pavlov — APPROVED** [review]
I don't think you need dots with the list of links, otherwise, looks great!

**Pull request merged**